index.html

```diff
@@ -118,32 +118,32 @@ <h2 class="project-title">Tonic</h2>
               </ul>
             </div>
             <div class="action">
-              <button type="button" class="action-button">See Project</button>
+              <button onclick="a(1)" id="btn-1" type="button" class="action-button">See Project</button>
             </div>
           </div>
         </div>
 
```

Comment:
- Kindly your work section is coming, which has to be created dynamically with the information from the projects array (all the HTML of the section is created when the page loads). This requires changes in existing code - already created HTML code for the projects list should be removed, and instead it should be added via JavaScript. from the static (HTML) page, 👍👌 kindly update through the requirement to make your page dynamic 👍👍

Reply:
> Correction

Remove the static template from the index file
Now the template is dynamic and is loaded from the script.

|After|Later|
|:------:|:------:|
|Index.html|card.js|
|![imagen](https://user-images.githubusercontent.com/90431831/217758850-1428aae3-6ab1-457b-accb-acdeb8bdafc9.png)|![imagen](https://user-images.githubusercontent.com/90431831/217758186-b6dd5004-c187-4843-ad1f-ad514bf4f612.png)|